Tests/Integration/CloudToDeviceMessageSizeLimitBaseTests.cs

```diff
@@ -3,31 +3,33 @@
 
 namespace LoRaWan.Tests.Integration
 {
-    using LoRaTools.LoRaPhysical;
+    using LoRaTools.LoRaMessage;
+    using LoRaWan.NetworkServer.BasicsStation;
     using LoRaWan.Tests.Common;
 
     public class CloudToDeviceMessageSizeLimitBaseTests : MessageProcessorTestBase
     {
-        public static Rxpk CreateUpstreamRxpk(bool isConfirmed, bool hasMacInUpstream, LoRaDataRate datr, SimulatedDevice simulatedDevice)
+        public static (RadioMetadata radioMetaData, LoRaPayload loraPayload) CreateUpstreamMessage(bool isConfirmed, bool hasMacInUpstream, LoRaDataRate datr, SimulatedDevice simulatedDevice)
```

Comment:
Suggestion:

```suggestion
        public static (RadioMetadata RadioMetadata, LoRaPayload LoRaPayload) CreateUpstreamMessage(bool isConfirmed, bool hasMacInUpstream, DataRate datr, SimulatedDevice simulatedDevice)
```

- Use Pascal casing for tuple element names.
- Use consistent casing between type and tuple element names.
- Change `datr` to be `DataRate`, which works for LoRa/FSK.